src/iterative_sorting/iterative_sorting.py

```diff
@@ -14,14 +19,18 @@ def selection_sort( arr ):
 
 
 
-
+# TO-DO:  implement the Bubble Sort function below
+def bubble_sort(arr):
+    swap_flag = True
+    while swap_flag:
+        swap_flag = False
+        for i in range(0, len(arr) - 1):
+            if arr[i] > arr[i + 1]:
+                arr[i], arr[i + 1] = arr[i + 1], arr[i]
+                swap_flag = True
     return arr
 
 
-# TO-DO:  implement the Bubble Sort function below
-def bubble_sort( arr ):
-
-    return arr
 
 
 # STRETCH: implement the Count Sort function below
```

Comment:
I know you're busy and it's definitely optional but you are OWNING the MVP, consider doing stretch sometimes! That's the obligatory "push the strong students", the other feedback I can give you is that your code looks super clean and easy to read, make sure not to fall out of the habit of leaving comments for more complicated code.